connectors-sdk/connectors_sdk/models/octi/knowledge/entities.py

```diff
@@ -93,12 +95,71 @@ def to_stix2_object(self) -> stix2.v21.Identity:
         )
 
 
+@MODEL_REGISTRY.register
+class Sector(BaseIdentifiedEntity):
+    """Represents a Sector on OpenCTI."""
+
+    name: str = Field(
+        description="Name of the sector.",
+        min_length=1,
+    )
+    description: Optional[str] = Field(
```

Comment:
not important : either we change it everywhere either we keep Optional, anyway it's another subject to discuss for unicity I guess
```suggestion
    description: str | None = Field(
```